.github/workflows/beam_PreCommit_Go.yml

```
@@ -19,10 +19,10 @@ on:
   push:
     tags: ['v*']
     branches: ['master', 'release-*']
-    paths: ['model/**', 'sdks/go.**', 'release/**', '.github/workflows/beam_PreCommit_Go.yml']
+    paths: ['model/**', 'sdks/go.mod', 'sdks/go.sum','sdks/go/**.go', 'release/**', '.github/workflows/beam_PreCommit_Go.yml']
```

```suggestion
    paths: ['model/**', 'sdks/go.mod', 'sdks/go.sum','sdks/go/**', 'release/**', '.github/workflows/beam_PreCommit_Go.yml']
```

I'd lean towards `/**` to pick up changes to gradle files, etc... here and elsewhere

---

I forgot about gradle files (and any other scripts). Those are important in this case. Fixed. Thanks!